src/psyclone/dynamo0p3.py

```diff
@@ -9457,7 +9120,7 @@ def ref_name(self, function_space=None):
                     "associated with this argument (fss='{1}')".format(
                         function_space.orig_name,
                         self.function_space_names))
-        if self._type == "gh_field":
+        if self._type in LFRicArgDescriptor.VALID_FIELD_NAMES:
             return "vspace"
         elif self.is_operator:
```

Comment:
Not your fault but since we're here, pylint points out that the `elif` on 9125 can be an `if` instead.

Reply:
Done (now line 9124 afer removal of `getkerneldescriptors` import above)